vispy/app/backends/_qt.py

```diff
@@ -230,6 +230,20 @@ def _vispy_get_native_app(self):
         # Return
         return app
 
+    def _vispy_sleep(self, duration_sec):
```

Comment:
usually we write in the base class that the method can be overridden by subclasses if they support a nicer functionality